CONTRIBUTING.md

```diff
@@ -23,10 +23,12 @@ The project's Gemfile contains a few developer tools to help keep the project ti
 Declare model attributes in the following order:
 
  1. includes (e.g. `include UserPasswordConcern`)
- 2. callbacks
- 3. attribute overrides (e.g. `enum`s)
- 4. validations
- 5. relations
+ 2. declarations (e.g. `acts_as :superclass`, `stampable`)
+ 3. callbacks
+ 4. attribute overrides (e.g. `enum`s, workflows)
+ 5. validations
+ 6. associations
```

Comment:
O I was looking for another para in the comments. XD  
What has it got to do with validations though?  Are you considering validations as a form of callback? Haha just trying to understand why placing associations above validations help with composability.